build/common/installer/scripts/tomlparser-common-agent-config.rb

```diff
@@ -11,6 +11,7 @@
 @disableTelemetry = false
 @logEnableKubernetesMetadataCacheTTLSeconds = 60
 @enableHighLogScaleMode = false
+@enableCustomMetrics = false
```

Comment:
Change looks good. Few questions
1. Did we test the custom metrics enablement and disablement works? 
2. how much perf improvements we are seeing when custom metrics disabled?
3. what's plan on deprecate the custom metrics for customers who will be enabling through config map settings?

Reply:
1. Tested it for legacy its working. MSI pending
2. Pending
3. After Aug, we will make configmap enablement a no-op